docs/api.md

```diff
@@ -406,6 +406,12 @@ if the url has a query parameter of `keys=true`.
   
   <!--end-request-body-post-accountcreate-metricsContext-->
 
+* `marketingOptIn`: *boolean*
+
+  <!--begin-request-body-post-accountcreate-marketingOptIn-->
+  
```

Comment:
But hey, I wouldn't have even thought to add this if it were for the pre-commit hook!

(Though, also, it was odd that after I had created the commit, I noticed that there was a new unstaged file, and amended my commit to add this file. Is there an earlier hook that makes sense?)